src/Store.js

```diff
@@ -54,7 +55,7 @@ class Store {
     this.access = options.accessController || defaultAccess
 
     // Create the operations log
-    this._oplog = new Log(this._ipfs, this.identity, { logId: this.id, access: this.access })
+    this._oplog = new Log(this._ipfs, this.identity, { logId: this.id, access: this.access, sortFn: options.sortFn })
```

Comment:
I think we're relying on a null check in `ipfs-log` to deliver the default sort function in the [`Log` constructor](https://github.com/orbitdb/ipfs-log/blob/master/src/log.js#L68). It's probably ok but maybe we want to be explicit?